pkg/trisa/api/v1beta1/errors.go

```diff
@@ -73,16 +73,15 @@ func Errorp(err error) (e *Error, ok bool) {
 		return e, ok
 	}
 
-	// TODO: DANGER! This doesn't work for some reason, the tests just hang?
-	// var s *status.Status
-	// if s, ok = status.FromError(err); ok {
-	// 	es := s.Details()
-	// 	if len(es) == 1 {
-	// 		if e, ok = es[0].(*Error); ok {
-	// 			return e, ok
-	// 		}
-	// 	}
-	// }
+	var s *status.Status
+	if s, ok = status.FromError(err); ok {
+		es := s.Details()
+		if len(es) == 1 {
+			if e, ok = es[0].(*Error); ok {
+				return e, ok
+			}
+		}
+	}
```

Comment:
Apparently this is no longer hanging?

Reply:
Yeh, I've noticed that too … something changed in the Google dependency for the better!